core/services/chainlink/cfgtest/dump/full-custom.env

```diff
@@ -33,6 +33,11 @@ DATABASE_LOCKING_MODE=advisory
 LEASE_LOCK_DURATION=5s
 LEASE_LOCK_REFRESH_INTERVAL=2s
 
+AUDIT_LOGGER_ENABLED=false
+AUDIT_LOGGER_FORWARD_TO_URL=http://localhost:9898
+AUDIT_LOGGER_JSON_WRAPPER_KEY=event
+AUDIT_LOGGER_HEADERS=
```

Comment:
This one needs an example value to validate the dump method will convert properly.